test/functional/test-amp-video.js

```diff
@@ -127,4 +146,85 @@ describe('amp-video', () => {
     }, sources)).to.be.rejectedWith(/start with/);
   });
 
+  it('should not set src or preload in prerender mode', () => {
+    return getVideo({
+      src: 'video.mp4',
+      width: 160,
+      height: 90,
+      'preload': 'auto',
```

Comment:
This is exactly the situation I was describing above. Please add another test where the 'preload' attribute will be absent.